### ISSUE TITLE
virtualenv exclude pre-installed by default

### ISSUE THREAD

**Issue by jayvdb** [indygreg/PyOxidizer#163]
The list of packages included by default in a virtualenv is known and should be excluded by default when building.

If excluding modules, they are: setuptools, pip, easy_install, pkg_resources, and usually wheel if built with `virtualenv` instead of `venv`.

**Comment by indygreg** [comment]
Yeah, I've noticed these packages getting picked up as well. I agree we should probably exclude them by default.
